### PR TITLE
systemd-sysusers: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-sysusers.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# systemd-sysusers
+
+> creates system users and groups.
```
Comment (kbdharun) [inline pages/linux/systemd-sysusers.md:3]:
```suggestion
> Create system users and groups.
```

File: pages/linux/systemd-sysusers.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+`systemd-sysusers {{config_pathpath}}`
+
+- Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:
+
+`systemd-sysusers --dry-run {{config_pathpath}}`
+
+- Print the contents of the specified configuration file:
+
+`systemd-sysusers --cat-config {{config_path}}`
+
```
Comment (kbdharun) [inline pages/linux/systemd-sysusers.md:17]:
```suggestion
```

File: pages/linux/systemd-sysusers.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# systemd-sysusers
+
+> creates system users and groups.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
+
+- Create Users and Groups from a Specific Configuration File:
```
Comment (kbdharun) [inline pages/linux/systemd-sysusers.md:6]:
```suggestion
- Create users and groups from a specific configuration file:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/systemd-sysusers.md (hunk 2 above)
Comment (quantumflo) [reply pages/linux/systemd-sysusers.md]: Do you want me to remove this command ?

Comment (kbdharun) [reply pages/linux/systemd-sysusers.md]: No, the extra line after the command.

Generally, you only need 1 line for having an end of line, any other additional one as added in code as blank lines, our linter doesn't allow these extra blank lines at last that's why I made this suggestion to remove it.

File: pages/linux/systemd-sysusers.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# systemd-sysusers
+
+> Creates system users and groups.
```
Comment (kbdharun) [inline pages/linux/systemd-sysusers.md:3]:
```suggestion
> Create system users and groups.
```

File: pages/linux/systemd-sysusers.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# systemd-sysusers
+
+> Create system users and groups.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
+
+- Create users and groups from a specific configuration file:
+
+`systemd-sysusers {{config_pathpath}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-sysusers.md:8]:
```suggestion
`systemd-sysusers {{path/to/file}}`
```

File: pages/linux/systemd-sysusers.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# systemd-sysusers
+
+> Create system users and groups.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
```
Comment (acuteenvy) [inline pages/linux/systemd-sysusers.md:4]:
```suggestion
> Create system users and groups.
> If the config file is not specified, files in the `sysusers.d` directories are used.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
```

File: pages/linux/systemd-sysusers.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+`systemd-sysusers {{config_pathpath}}`
+
+- Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:
+
+`systemd-sysusers --dry-run {{config_pathpath}}`
+
+- Print the contents of the specified configuration file:
+
+`systemd-sysusers --cat-config {{config_path}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-sysusers.md:16]: This doesn't work, the path is ignored.
If you want to do this, you can just use `cat` instead.

```suggestion
- Print the contents of all config files (before each file, its name is printed as a comment):

`systemd-sysusers --cat-config`
```

File: pages/linux/systemd-sysusers.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+> Create system users and groups.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
+
+- Create users and groups from a specific configuration file:
+
+`systemd-sysusers {{config_pathpath}}`
+
+- Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:
+
+`systemd-sysusers --dry-run {{config_pathpath}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-sysusers.md:12]:
```suggestion
- Process configuration files and print what would be done without actually doing anything:

`systemd-sysusers --dry-run {{path/to/file}}`
```

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**